src/HelixProject.js

```diff
@@ -171,16 +179,16 @@ class HelixProject {
   }
 
   async checkPaths() {
-    const idxPath = path.resolve(this._cwd, INDEX_MD);
```

Comment:
I would just guard it with an if. so you can set it from external

Reply:
I do understand what you mean. I did not change that code (and do not want to), I just swapped the order of execution.